includes/admin/class-wcs-admin-post-types.php

```diff
@@ -249,7 +249,12 @@ public function print_bulk_actions_script() {
 
 		$post_status = ( isset( $_GET['post_status'] ) ) ? $_GET['post_status'] : '';
 
-		if ( 'shop_subscription' !== get_post_type() || in_array( $post_status, array( 'cancelled', 'trash', 'wc-expired' ) ) ) {
+		$subscription_id = $GLOBALS['post'] ?? '';
```

Comment:
I haven't tested this but `$GLOBALS['post']` is the post object isn't it? whereas we want to send the ID to the data store function

Does that mean we want this line need to be something like:

```
$subscription_id = ! empty( $GLOBALS['post']->ID ) ? $GLOBALS['post']->ID : '';
```

Reply:
Ah you're right. When I debugged, it did show up as a post object and it worked because `WC_Data_Store::load( 'subscription' )->get_order_type()` can take object just fine too.

Addressed in [e9f34d1](https://github.com/Automattic/woocommerce-subscriptions-core/pull/318/commits/e9f34d1e6d4e20971ec5b36f11b192453f2d532f).